presto-spi/src/main/java/com/facebook/presto/spi/security/Privilege.java

```diff
@@ -15,5 +15,5 @@
 
 public enum Privilege
 {
-    SELECT, DELETE, INSERT
+    SELECT, DELETE, INSERT, UPDATE
```

Comment:
Is it possible to have an `UDATE` grant?  If not, I'd leave it out for now since we can't test it.